akka-persistence-typed/src/test/scala/akka/persistence/typed/scaladsl/EventSourcedBehaviorSpec.scala

```diff
@@ -762,31 +762,30 @@ class EventSourcedBehaviorSpec extends ScalaTestWithActorTestKit(EventSourcedBeh
           ctx ⇒
             counterWithSnapshotAndRetentionProbe(ctx, pid, snapshotProbe.ref, retentionProbe.ref)
               .snapshotEvery(2)
-              .withRetention(
-                RetentionCriteria(snapshotEveryNEvents = 2, keepNSnapshots = 2, deleteEventsOnSnapshot = false))))
+              .withRetention(RetentionCriteria(snapshotEveryNEvents = 3, keepNSnapshots = 2))))
```

Comment:
@patriknw this still passes oddly?

Reply:
Still investigating.

Reply:
Note snapshotEvery(2) vs snapshotEveryNEvents = 3

They must be the same, we have another issue for that duplication in the api.

Reply:
Ok, that duplication is one thing I was unsure of. Given sharding has that usage of snapshotEvery in it's delete usage.